test/e2e/llmisvc/test_llm_inference_service.py

```diff
@@ -434,7 +444,35 @@ def delete_llmisvc(kserve_client: KServeClient, llm_isvc: V1alpha1LLMInferenceSe
             KSERVE_PLURAL_LLMINFERENCESERVICE,
             llm_isvc.metadata.name,
         )
-        print(f"✅ LLM inference service {llm_isvc.metadata.name} deleted successfully")
+        print(f"✅ LLM inference service {llm_isvc.metadata.name} deletion initiated")
+
+        # Wait for the resource to be fully deleted to avoid HTTP 409 conflicts
+        # when the next test tries to create a resource with the same name
```

Comment:
I think tests should create different names, maybe what we're missing is the addition of the api version in the service name?